.github/workflows/actions/deploy/action.yml

```diff
@@ -108,7 +108,7 @@ runs:
 
       - uses: hashicorp/setup-terraform@v1.3.2
```

Comment:
Could we update the action to v2.0.0 as below?

Reply:
Done